lib/router.coffee

```diff
@@ -76,7 +76,13 @@ Router.map ->
         filter = @params.query
         if filter.search
           filter.$text = { $search: filter.search }
-          delete filter.search
+        delete filter.search  
```

Comment:
Did this mean to make it outside the `if filter.search` block?  It should be okay either way -- if it doesn't exist, the `delete` will just be a no-op.